src/environments/array.js

```diff
@@ -194,8 +194,8 @@ const htmlBuilder = function(group, options) {
         hlinePos.push(totalHeight);
     }
 
-    for (r = 0; r < groupValue.body.length; ++r) {
-        const inrow = groupValue.body[r];
+    for (r = 0; r < group.value.body.length; ++r) {
+        const inrow = group.value.body[r];
```

Comment:
These are the ones I missed in my PR?

Reply:
No. This is a consequence of me removing
```
const groupValue = assertNodeType(group, "array").value;
```
since it should not be needed. `htmlBuilder`'s `group` parameter is already strictly typed as `ParseNode<"array">`.